Roslyn.sln

```diff
@@ -382,6 +382,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RemoteWorkspaces", "src\Wor
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ServiceHub", "src\Workspaces\Remote\ServiceHub\ServiceHub.csproj", "{80FDDD00-9393-47F7-8BAF-7E87CE011068}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RepoUtil", "src\Tools\RepoUtil\RepoUtil.csproj", "{1CA184D3-89CB-4074-BEC5-F8AEBA657D41}"
```

Comment:
We'll have a bunch of tools and .targets files that are specific to the structure of Roslyn-like repos (test impact, DSR, etc.). They are not general tools that anybody can use for any git repo or project build system shape. However, any repo that opts into our conventions would benefit from these. I'd collect such tools in "roslyn-tools" repo.